src/codeu/chat/server/bots/HangmanBot.java

```diff
@@ -0,0 +1,68 @@
+package codeu.chat.server.bots;
+
+public class HangmanBot extends Bot {
+
+	private String word;
+	private String guessString = "";
+	private int lives = 6;
+
+	public String loseOrRejectGuessWith(String message) {
+		lives -= 1;
+		if (lives < 0) {
+			word = null;
+			return "Out of lives! Sorry but you lose! The word was " + word;
+		}
+		return message;
+	}
+
+	public String reactTo(String command, String[] args) {
```

Comment:
Does this override something from the `Bot` class? Might add an `@Override` annotation to make it obvious. 

Does a Bot know **who** sent the message? If not, does that mean two people can be playing at the same time? How does that work?

Is it always a command, or can it just be text as well?